src/wp-admin/includes/deprecated.php

```diff
@@ -1,6 +1,6 @@
 <?php
 /**
- * Deprecated admin functions from past ClassicPress versions. You shouldn't use these
+ * Deprecated admin functions from past WordPress versions. You shouldn't use these
```

Comment:
I would have said "WordPress or ClassicPress" here, but it could be argued either way.  Do you think we should change this now, or wait until we start to deprecate functions?  I think we could easily forget at that point.

Reply:
On a closer look, I do think we should change this.  We are already doing the same thing elsewhere (except I used "ClassicPress or WordPress"):

https://github.com/ClassicPress/ClassicPress/blob/6a6ce589f760851f10654349b9a181091f495642/src/wp-includes/functions.php#L3826-L3829

Reply:
The thought behind it is that functions that are deprecated in WP, are native to WP and have nothing to do with CP. When CP 1.0.0 is released these deprecated functions still belong to older WP versions and therefore _Deprecated admin functions from past WordPress versions._ 
What do you think?

Reply:
I was thinking that we may also deprecate functions in CP.

I don't have a strong opinion about this, but let's be consistent either way:  if we decide to leave these as WP, change the existing "ClassicPress or WordPress" text back to "WordPress".  I'm also fine with doing that in a separate PR since I was the one who put those in there in the first place.

Reply:
> I was thinking that we may also deprecate functions in CP.

If we're using the same function-name/file for that, then I agree that I should change it to _ClassicPress or WordPress_